doc/conf_ref.rst

```diff
@@ -525,7 +525,21 @@ configuration file by your distribution to override the DNF defaults.
     ``"/etc/dnf/vars", "/etc/yum/vars"``. See :ref:`variable files <varfiles-label>`
     in Configuration reference.
 
-.. _conf_repo_options-label:
+.. _exclude_from_weak-label:
```

Comment:
The other options are sorted alphabetically. Can you please move these to the correct place?

Reply:
Thanks, the problem was created by rename of options.